pcsx2/gui/AppMain.cpp

```diff
@@ -1152,6 +1152,9 @@ void Pcsx2App::SysExecute()
 void Pcsx2App::SysExecute( CDVD_SourceType cdvdsrc, const wxString& elf_override )
 {
 	SysExecutorThread.PostEvent( new SysExecEvent_Execute(cdvdsrc, elf_override) );
+#ifndef DISABLE_RECORDING
+	RecordingReset();
+#endif
```

Comment:
if my suggestion is taken, i imagine this would change to `g_InputRecording.RecordingReset()`